website/site/content/docs/configuration-options.md

```diff
@@ -60,6 +60,19 @@ public_folder: "/images/uploads"
 
 Based on the settings above, if a user used an image widget field called `avatar` to upload and select an image called `philosoraptor.png`, the image would be saved to the repository at `/static/images/uploads/philosoraptor.png`, and the `avatar` field for the file would be set to `/images/uploads/philosoraptor.png`.
 
+## Slug Type
+
+By default, filenames (slugs) for entries created in the CMS are sanitized according to [RFC3987](https://tools.ietf.org/html/rfc3987) and the [WHATWG URL spec](https://url.spec.whatwg.org/). This spec allows non-ASCII (or non-Latin) characters to exist in URLs. However, for maximum compatibility, you can also set a different slugification option:
+
+- `iri` (default): Keeps Unicode characters in slugs, according the the IRI draft spec ([RFC3987](https://tools.ietf.org/html/rfc3987)) and the [WHATWG URL spec](https://url.spec.whatwg.org/).
+- `latin`: Removes accents/diacritics from slug, then strips out all non-valid URL characters and periods (see `ascii` below).
+- `ascii`: Strips out all characters except valid URI chars (RFC3986) or periods (0-9, a-z, A-Z, `_`, `-`, `~`).
```

Comment:
If `latin` is effectively `ascii` + accent removal, it'd be clearer to just state that. Also, we should order these options from most to least permissive, which just mean swapping the last two.

Reply:
> If `latin` is effectively `ascii` + accent removal, it'd be clearer to just state that.

Agreed, can you suggest wording?

> Also, we should order these options from most to least permissive, which just mean swapping the last two.

I originally ordered them in that way, but I'm not sure. You're going to end up with *more* chars left in `latin` than in a pure RFC3986 format (`ascii` option), since you're converting accented chars instead of just stripping them out.

Reply:
> since you're converting accented chars instead of just stripping them out.

Then why does it not read:

```
Converts accent/diacritic characters from slug to ASCII equivalents, then strips out all non-valid URL characters and periods (see `ascii` below).
```

?

Reply:
@rdela That is what it does -- my wording needs to be cleared up there.

Reply:
Maybe they are in the right order after all and `latin` just needs better description like example suggestion above?

Reply:
Yes, exactly.

Reply:
Think boils down to even though `diacritics.remove` https://github.com/netlify/netlify-cms/pull/1135/files?diff=split#diff-32295f723473b0263bfa351b7ffab64eR75 seems unfortunately named https://www.npmjs.com/package/diacritics we should use “convert” or “swap” in our docs

Reply:
maybe even leave a comment in the source to avoid future confusion

Reply:
I can definitely see the confusion there, but it does make sense in that it is removing the diacritics and leaving the base chars intact. I'll definitely clarify the comments and docs.

Reply:
“replace” could also work as synonym for “remove” https://github.com/andrewrk/node-diacritics/blob/master/index.js#L310

Reply:
```
Replaces accent/diacritic characters from slug with ASCII equivalents, then strips out all non-valid URL characters and periods (see `ascii` below).
```

Reply:
> it does make sense in that it is removing the diacritics and leaving the base chars intact

Oh yeah I can see that meaning now

Reply:
Starting to wonder if we should do this a little different. This boils down to URI/3986 or IRI/3987, with the option to strip accents. Maybe:

```yaml
slug: iri

# or

slug:
  - protocol: iri
  - strip_accents: true
```

Would stripping accents ever make sense when using IRI protocol?

Reply:
@erquhart Actually, I think that's the best option. Stripping accents is really going to be independent of the format.

Reply:
Cool, so most folks can just do `slug: uri` or `slug: iri`, but others can provide object if they need tighter control.

Btw I love that pattern and would love to see us using it as we expand and restructure the APIs - the pattern being config options accepting a primitive value for simplicity, but also an object for fine-grained settings.